### PR TITLE
[FEATURE] Replaces CRON usages by a proper systemd timer

### LINKED ISSUE #77: README.md for Arch
I've just set this up on a couple of old Cubox 1's running Arch Linux. If you could add onto the README.md that Arch uses systemd/Timers by default but installing a cronie as a package 'pacman -S cronie' followed by 'systemctl enable cronie', adds cron timers. All works fine.
Thanks for the useful utility.

### LINKED ISSUE #84: Use systemd timers instead of cron?
Systemd has timers which can be an alternative to cron jobs. Thoughts on moving to timers and no longer needing a dependency on cron?

### PR DESCRIPTION
**Pull request by HorlogeSkynet**
This patch additionally adds out-of-the-box compatibility for Arch-based distributions.

> Closes #84
> Closes #77

Bye :wave:

### CODE REVIEW

**Comment by azlux** [comment]
That's an interesting pull requests. Thanks, I will take a look.

**Comment by emwe1** [comment]
I found this after I was creating my own solution for Manjaro Linux, which has per default no cron installed. Here are my changes.

[changedfiles.tar.gz](https://github.com/azlux/log2ram/files/4907670/changedfiles.tar.gz)

File: uninstall.sh [hunk 1]
```diff
@@ -2,12 +2,11 @@
 
 if [ "$(id -u)" -eq 0 ]
 then
-  service log2ram stop
-  systemctl disable log2ram
-  rm /etc/systemd/system/log2ram.service
+  systemctl stop log2ram.service log2ram-daily.timer
+  systemctl disable log2ram.service log2ram-daily.timer
+  rm -rf /etc/systemd/system/log2ram*
```
Comment (azlux) [inline uninstall.sh:7]: can we do better than a wildcard ? It's only for 2 know file here.

**Comment by azlux** [comment]
@emwe1 I've read yours files, it's almost the same as @HorlogeSkynet . If you read them I think you will be aggre to merge this PR too.

@HorlogeSkynet If you check the @emwe1, one thing is added compared to your PR, it check if `/etc/logrotate.d` exist before install the logrotate file. What do you think ?
For the PR, I've write a small minor comment you can fix, but everything else is good for me.

File: uninstall.sh (hunk 1 above)
Comment (HorlogeSkynet) [reply uninstall.sh:13]: Actually, potentially 3.
If you happen to run `systemctl edit log2ram-daily.timer` as I've added in the README, a file named `/etc/systemd/system/log2ram.timer.d/override.conf` would be created containing the user configuration.
The wildcard has been chosen to remove **any** trace of the service from a systemd PoV, including such an overriding.

Tell me what you think about that :ok_hand:

**Comment by HorlogeSkynet** [comment]
Some of @emwe1's changes have been included, as long as minor stuffs. Keep me posted :wave:

**Comment by azlux** [comment]
That was fast. thank you
Look good for me.

**Pull request merged**